### PR TITLE
cdk 5.0-20200923

### PR DESCRIPTION
**Pull request by mprince**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/cdk.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class Cdk < Formula
   desc "Curses development kit provides predefined curses widget for apps"
   homepage "https://invisible-island.net/cdk/"
-  url "https://invisible-mirror.net/archives/cdk/cdk-5.0-20200228.tgz"
-  version "5.0.20200228"
```
Comment (SMillerDev) [inline Formula/cdk.rb:5]: Does it still detect the correct version?

Comment (mprince) [reply Formula/cdk.rb:5]:
```sh
$ brew livecheck cdk --debug
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/cdk.rb

Formula:          cdk
Livecheckable?:   Yes

URL:              https://invisible-mirror.net/archives/cdk/
Strategy:         PageMatch
Regex:            /href=.*?cdk[._-]v?(\d+(?:[.-]\d+)+)\.t/i

Matched Versions:
5.0-20060507, 5.0-20081105, 5.0-20090215, 5.0-20110517, 5.0-20120318, 5.0-20120323, 5.0-20130901, 5.0-20131107, 5.0-20140101, 5.0-20140118, 5.0-20141106, 5.0-20150928, 5.0-20160131, 5.0-20161120, 5.0-20161204, 5.0-20161210, 5.0-20171209, 5.0-20180306, 5.0-20190220, 5.0-20190224, 5.0-20190303, 5.0-20191231, 5.0-20200228, 5.0-20200923
cdk : 5.0-20200923 ==> 5.0-20200923
```

**Comment by mprince** [comment]
[Licensing](https://invisible-island.net/cdk/#licensing) is very confusing

File: Formula/cdk.rb (hunk 1 above)
Comment (issyl0) [reply Formula/cdk.rb:5]: Yes:

```
issyl0 in ophiuchus in /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core on bump-cdk-5.0-20200923
➜ cdk5-config --version
5.0.20200923
```

**Comment by issyl0** [comment]
I rebased this PR on the latest `master` so CI builds bottles for the new macOS Big Sur as well as the other versions.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**